pandas/tools/tests/test_pivot.py

```diff
@@ -92,7 +91,11 @@ def test_pivot_table_dropna(self):
                                     (u('C'), u('b')),
                                     (u('C'), u('c')),
                                     (u('C'), u('d'))])
-
+    
+        assert_equal(pv_col.index.names, names1)
+        assert_equal(pv_ind.columns.names, names1)
```

Comment:
@jreback do you mean i need to fully rewrite `test_pivot_table_dropna` or just make a separate DataFrame for this case?